tests/drivers/build_all/gpio/testcase.yaml

```diff
@@ -11,3 +11,9 @@ tests:
       - gpio
       - spi
       - i2c
+
+  drivers.gpio.build.efinix_sapphire:
+    min_ram: 32
+    platform_allow: titanium_ti60_f225
+    depends_on: gpio
+    extra_args: DTC_OVERLAY_FILE="efinix_sapphire.overlay"
```

Comment:
this doesn't look right to me

Reply:
@cfriedt this is what you meant with that comment?

Reply:
@cfriedt pinging. Can you let us know?

Reply:
@cfriedt @carlocaione How can we find a solution? The PR has been stagnant :( For some time

Reply:
@npitre @galak  would you be able to do a review and +1?

Reply:
@cfriedt ping. Please review and let me know if any changes is required.